CONTRIBUTING.rst

```diff
@@ -351,3 +351,29 @@ usages of the modified APIs in guides and examples.
 If adding a new public Python module, create a corresponding doc file for the module under
 ``docs/source/python_api`` - `see here <https://github.com/mlflow/mlflow/blob/v0.9.1/docs/source/python_api/mlflow.tracking.rst#mlflowtracking>`_
 for an example.
+
+
+Writing an MLflow Plugin
```

Comment:
Can we add a table of contents to the `CONTRIBUTING.rst` document to more effectively surface the fact that this info exists? I'm not sure if users will naturally scroll to the bottom of the document when looking for plugin info.